### PR TITLE
Early Crafted Bow Renaming and Description Editing

### PR DESCRIPTION
**Pull request by Custard-Mouse**
#### Summary
None

#### Purpose of change
While I was peaking around at the past archery PRs and documentation, it seems a number of players were frustrated by the descriptions and names of the weapons they had not really matching their in game performance. It seems like these weapons as they fit in resource wise/skill wise are most likely in the right place mechanically, but I hope that more direct descriptions of the type of bow, and the intended capabilities of it will set the player expectations for these weapons at a appropriate level.

#### Describe the solution
Renamed and tuned the description on the three earliest craftable bows, previously the "survival bow", "short bow", and "long bow", they are now respectively named the "makeshift self bow", "short self bow", and "self bow". 

The "makeshift self bow" now describes it's intended mechanical purpose, it's a practice tool, what someone with no experience as a archer or bowyer would be able to make, while it can be technically be used in combat it's efficacy against anything that would let the player within it's short range is close to 0. 

The "short self bow" now describes the form factors inherent inefficiency to some degree. It also clears up that the bow is, "...suitable for... harassing unarmored enemies" hopefully this alerts players to the fact that this still isn't going to be a horde clearer, it's a weapon that can be pressed into a skirmishing role but is more still a stepping stone for the bows that can be seen as both effective combat and hunting weapons.

The "self bow" again adds a little more to specify as to what the player is actually receiving weapon wise, while attempting to distance itself slightly from describing itself as a true English longbow, this is mostly due to the association folks will have with the concept of the English longbow and what was militarily fielded by English longbowmen. I tried to keep the description as short as I could but still wanted to communicate what the player could expect this weapon would look like in real life, and it's combat capabilities, being one of the first bows that a player might find success using without having to lean heavily on skirmishing tactics.

#### Describe alternatives you've considered
Removing the "self" prefix in the new names for these bows: I'm adding the self prefix to these weapons because reasonably that's what they all are, single wooden stave bows. I also hope the niche nomenclature for many players should help to prevent them from establishing expectations based on the previous names they held, from what I can tell reading over the previous PRs these weapons are supposed to mechanically be rather crude attempts at making bows, rather then being something that should be compared to a properly cured, tillered, and weighted short or long bows, though admittedly more developed bows can certainly be built in a manner that would perform similarly to either the "short self bow" or "self bow".

Changing the "self" prefix in the new names to something along the lines of "makeshift" or "crude": This is a alternative I find there could be a very convincing argument for based on my understanding of what these bows should do mechanically, my worry would be the gap left in terms of how the player perceives bow technology in game, eventually I'd like to get around to flushing out some more tiering and variety to bow recipes, as well as adding proper curing, but until then the "short self bow" and "self bow" are in this weird space of being bows that could be made through very crude methods, but effectively describe themselves in a way that leaves enough nuance for the player to see these as being developed bows, hopefully preventing that gap from being noticed.

Different names and or some more specification and the descriptions: Specifically I entertained the idea of naming the "makeshift self bow" as a "practice bow" or "practice self bow" but felt like these names didn't imply just how much of a piece of junk this thing should be and is. The other idea was to specify with the "self bow" in its description that it is similar to some of the lighter English longbow designs, but was worried this extra string length would clutter the weapons description too much and I hope that the niche nomenclature of "self bow" along with the expected capabilities of this weapon being included in the description clear things up enough.

Not changing the name for the weapons but descriptively indicating their intended abilities better: This is another alternative that reasonably could be as valid as what names this PR currently has to offer, as the previous alternative mentioned these weapons exist in a weird space where recipe wise they are extremely crude bows yet mechanically bows with this sort of performance are most likely possible to craft through both crude and developed methods, my primary contention here would most likely be keeping "Longbow" as a name for that weapon, I believe there's a strong association most folks familiar with medieval history or archery have with the idea of a "Longbow" which translates usually to a expectation of what something like a 150 lbs draw weight Warbow could achieve, which for this item mechanically doesn't end up leading to healthy expectations of it's performance.

Not changing the description but finding more appropriate names: I can't think of any methods in which a pure name change would both be able to set reasonable expectations for what a bow should do mechanically yet doesn't end up contrasting it's own description in a strange way.

#### Testing
During each iteration of me editing and rethinking name changes I loaded CDDA up and verified with a new character that the names and descriptions for these weapons were changed using the debug spawn item menu.

### CODE REVIEW

**Comment by Maleclypse** [comment]
unrelated failure. rng based

**Review by Maleclypse — APPROVED** [review]
Thanks!

File: data/json/items/ranged/archery.json [hunk 1]
```diff
@@ -373,8 +373,8 @@
     "type": "GUN",
     "symbol": "(",
     "color": "yellow",
-    "name": { "str": "survival bow" },
-    "description": "A weak bow crudely crafted from a single uncured wood stave.  Weak and wildly inaccurate, but easy to make.",
+    "name": { "str": "makeshift self bow" },
+    "description": "A weak practice bow crudely crafted from a single uncured wood stave.  Not a effective weapon in any manner, but you have to start somewhere.",
```
Comment (schoolcf) [inline data/json/items/ranged/archery.json:377]:
```suggestion
    "description": "A weak practice bow crudely crafted from a single uncured wood stave.  Not an effective weapon in any manner, but you have to start somewhere.",
```

Comment (Custard-Mouse) [reply data/json/items/ranged/archery.json]: Thank you!

**Comment by kevingranade** [comment]
I'm really not a fan of the self bow nomenclature, it doesn't indicate anything meaningful about the bow, in particular it's not making any statement whatsoever about quality, it's just throwing in a detail of the bows construction.

TBH, with the exception of longbow being technically incorrect, I'm not really seeing a problem in the first place.

If survival bow is a problem, we can just stick makeshift in there instead or in addition, but I'm honestly not buying that it's a big problem, it feels more like, "make good bows easier to make" than a real confusion.

Backing up a bit, the *most basic* set of bows is outlined in this comment https://github.com/CleverRaven/Cataclysm-DDA/issues/39161#issuecomment-608008788

| Name | Craftable | Lootable | Notes | Current name |
|-----|-----|-----|-----|-----|
| Makeshift bow | yes | no | Cylindrical, straight bow | Survival Bow |
| Crude Bow | yes | no | Flatbow, maybe a little recurve | Longbow |
| Crude Horn Bow | yes | no | Composite bow fireable from tight quarters, very difficult to make | Shortbow |
| Wood Greatbow | yes | no | Just a super oversized wood reflex bow | Wood Greatbow |
| Compound Greatbow | yes | no | An oversized steel spring compound bow | Compound Greatbow |
| Modern Recurve  Bow | no | yes | Heavily recurved, fiberglass or other resin | Modern Recurve Bow |
| Compound Hunting Bow | no | yes | Modern resin compound bow | Compound Hunting Bow |

The main change here is that we're simply enforcing a hard line between craftable and lootable bows, everything is one or the other with zero overlap.  Shortbow stops being lootable because they're exceptionally rare, and we double down on it being a horn-and-sinew composite bow, longbows stop being lootable and stop being longbows.

If you want to fill in more, for example a traditional wood flatbow with higher quality, that's something that might be both lootable and craftable as they're not quite as rare as composite bows.

**Comment by kevingranade** [comment]
>  I'd like to do some standardization and add some meaningful details for bow's later on based on the construction/geometry they have,

It's hard to be more specific since you're not saying what kind of naming changes you're planning on making, but we're not going back to "reflex bow" vs "recurve bow" vs "reflex recurve bow".  In practice none of that matters, what matters is how heavy the draw weight is, and how efficient the bow is, which is a property of different categories of bow, but in practice it boils down to two flavors of modern bow and a few different flavors of craftable bow as outlined in my comment above.

We need to sort this out, because reading between the lines I'm concerned you're headed in a direction that doesn't really work. I can't imagine a scenario where self bow is a meaningful label to put on anything.

> it's more so that as of now the shortbow and longbow are locked at a relatively low draw-weight compared to what weights these weapons had when deployed in combat in the past

Of course they're not, they aren't war bows.
I'd estimate a combat shortbow is not particularly reasonable for the player to make, that's a very complex composite bow, and the longbow is not restricted to low draw, that's the wood greatbow. So I'm not really following what you're saying.

> Since I've already begun work on the backend mechanics for bows and plan to try and use the systems I'm establishing there to add in a few bow technologies and hopefully varied draw weights, should a naming update of any sort hold off till then?

No, we need to sort this out now, if you're making changes with some future thing in mind, you're going to need to outline what that future thing is, and it's far better to spend some time doing that now than working towards it without a clear idea of what it is.

**Comment by kevingranade** [comment]
Not the direction I want to go as written, see https://github.com/CleverRaven/Cataclysm-DDA/pull/55252#issuecomment-1036845754 for similar changes that would be accepted.

**Pull request closed without merging**